test/docker/Dockerfile.centos8

```diff
@@ -5,5 +5,9 @@ RUN sed -i 's/enabled=0/enabled=1/g' /etc/yum.repos.d/CentOS-Linux-PowerTools.re
 RUN yum install -y \
         python3-pip \
         libgomp \
-        mesa-libGL
+        mesa-libGL \
+        wget
 RUN python3 -m pip install --upgrade pip
+RUN wget https://repo.anaconda.com/archive/Anaconda3-2021.04-Linux-aarch64.sh -O ~/anaconda.sh
```

Comment:
Instead of adding `wget` as a dependency, you can use the `ADD` directive in the dockerfile. This will also allow docker to more intelligently update the container when the Anaconda installer changes.
```
ADD https://repo.anaconda.com/archive/Anaconda3-2021.04-Linux-aarch64.sh /root/anaconda.sh
```
https://docs.docker.com/engine/reference/builder/#add

You can make this change in each dockerfile.